### PR TITLE
Feature 90: Tooltip component

### PR DESCRIPTION
**Pull request by alexandre-lelain**
- Add Tooltip component.

Issue: [#90](https://github.com/attineos/atti-components/issues/90)

### CODE REVIEW

**Comment by callain** [comment]
Can you add the component in the quick demo part of the documentation ?

File: src/components/Tooltip/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Tooltip.defaultProps = {
+  position: 'top',
+}
+
+Tooltip.propTypes = {
+  /* The children on which the tooltip will be displayed over. */
+  children: PropTypes.oneOfType([PropTypes.element, PropTypes.node]).isRequired,
+  /* The message to display in the tooltip. */
+  message: PropTypes.string.isRequired,
```
Comment (akaala) [inline src/components/Tooltip/index.js]: Are you sure we should restrict it to string only ? What about having some DOM in it for example ?

File: src/components/Tooltip/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  onShowTooltip = () => this.setState({ showTooltip: true })
+
+  onHideTooltip = () => this.setState({ showTooltip: false })
+
+  render() {
+    const { children, message, position } = this.props
+    const { showTooltip } = this.state
+
+    return (
+      <Container onMouseLeave={this.onHideTooltip}>
```
Comment (akaala) [inline src/components/Tooltip/index.js]: So this component doesn't really work on mobile ? What about adding some support onClick for example ? :)

**Comment by alexandre-lelain** [comment]
The component was added to the demo @callain 👍

File: src/components/Tooltip/index.js (hunk 1 above)
Comment (alexandre-lelain) [reply src/components/Tooltip/index.js]: Good point. I added the option to pass a dom element

File: src/components/Tooltip/index.js (hunk 2 above)
Comment (alexandre-lelain) [reply src/components/Tooltip/index.js]: It does work on mobile thanks to `onFocus` prop :)

Comment (atrakeur) [reply src/components/Tooltip/index.js]: Confirmed working on https://deploy-preview-100--atti-components.netlify.com on android/FF

Seems good to me as it is.

**Pull request merged**